lib/segment/src/data_types/vectors.rs

```diff
@@ -11,7 +11,8 @@ use crate::vector_storage::query::context_query::ContextQuery;
 use crate::vector_storage::query::discovery_query::DiscoveryQuery;
 use crate::vector_storage::query::reco_query::RecoQuery;
 
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, JsonSchema)]
```

Comment:
Needs to be added to `schema_generator.rs`

Reply:
fixed